scrapper.py

```diff
@@ -141,10 +157,26 @@ def validate_config(crawler_path):
     return seed_urls, max_articles
 
 
-#  crawler = Crawler(validate_config(CRAWLER_CONFIG_PATH)[0],
-#  validate_config(CRAWLER_CONFIG_PATH)[1])
-
-
 if __name__ == '__main__':
-    # YOUR CODE HERE
-    pass
+    print('Validating config...')
+    valid_seed_urls, number_of_articles = validate_config(CRAWLER_CONFIG_PATH)
+    if valid_seed_urls and number_of_articles:
+        print(f'Config is correct.\nURLs found: {valid_seed_urls};\t'
+              f'Number of articles to parse: {number_of_articles}')
+        print('Preparing environment...')
+        prepare_environment(ASSETS_PATH)
+
+        print('Creating a crawler instance...')
+        crawler = Crawler(valid_seed_urls, number_of_articles)
+        crawler.find_articles()
+
+        print('Parsing...')
+        for article_id, article_url in enumerate(crawler.urls):
+            article_parser = HTMLParser(article_url, article_id + 1)
+            article = article_parser.parse()
+            article.save_raw()
+            print(f'Article {article_parser.article_id} of {number_of_articles} is done.')
+            if article_parser.article_id == number_of_articles:
```

Comment:
do you expect that `crawler.urls` may contain more articles than needed?

Reply:
Yes, I want to save and parse exactly as many articles, as it is specified in _config_ file

Reply:
It is what is expected from you, right. However, judging by your `Crawler.find_articles`, crawler.urls cannot contain more articles than needed, so why perform extra checks? 
Let me know if I miss something

Reply:
Yes, that's right. I forgot about it. Replaced with another check